L1Trigger/L1THGCal/python/hgcalBackEndLayer2Producer_cfi.py

```diff
@@ -82,13 +88,23 @@
                                nBins_X1_histo_multicluster=cms.uint32(42), # bin size of about 0.012
                                nBins_X2_histo_multicluster=cms.uint32(216), # bin size of about 0.029
                                binSumsHisto=binSums,
+                               kROverZMin=cms.double(0.076),
+                               kROverZMax=cms.double(0.58),
                                threshold_histo_multicluster=cms.double(10.),
                                neighbour_weights=neighbour_weights_1stOrder,
                                seed_position=cms.string("TCWeighted"),#BinCentre, TCWeighted
                                seeding_space=cms.string("RPhi"),# RPhi, XY
                                seed_smoothing_ecal=seed_smoothing_ecal,
                                seed_smoothing_hcal=seed_smoothing_hcal,
-                               )
+                              )
+
+## Note: this customization change both HGC and HFnose, to be revised
```

Comment:
should this be addressed within this PR?

Reply:
I have discussed with @jbsauvan 
The plan is to duplicate the full L1 sequence instead of having one single object with both HGC and HFnose TP.

I intended to do in subsequent PR ; is it ok ?

Reply:
The duplication of the sequence for the HFNose might not be straightforward. In any case it will require checks and validation. So I would prefer to keep this for an other PR.